poi-scratchpad/src/main/java/org/apache/poi/hssf/converter/ExcelToHtmlConverter.java

```diff
@@ -668,7 +668,7 @@ protected void processSheetHeader(Element htmlBody, HSSFSheet sheet) {
         htmlBody.appendChild(h2);
     }
 
-    public void processWorkbook(HSSFWorkbook workbook) {
+    public void processWorkbookSheet(HSSFWorkbook workbook) {
```

Comment:
This doesn't need to be renamed

Reply:
Yes, naming is not an issue. It was just my idea. Keeping the name same is also fine